templates/crd-servicedefaults.yaml

```diff
@@ -2,16 +2,15 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.5
+    controller-gen.kubebuilder.io/version: v0.3.0
```

Comment:
The updated version of `controller-runtime` updated the version of `controller-gen` causing the changes in this file that are purely cosmetic.